### PR TITLE
Add example for CSS pointer-events property

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds an example for [`pointer-events`](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events). Let me know if you want to see any changes. Thanks!

### CODE REVIEW

**Comment by wbamberg** [comment]
This is a nice example! I appreciate you including the SVG-specific values. What do you think about making the stroke wider, to make the difference more obvious:

screen shot 2018-02-26 at 1 51 19 pm

**Comment by ddbeck** [comment]
I tried that initially because you're right, it is more obvious for the `stroke` value. But it makes things a bit odd for the `fill` value. Apparently, the stroke is drawn from the edge of the circle both inward and outward, so the cursor changes halfway through the stroke, like this:

svg_slow

I'm not sure which value we ought to optimize for, to be honest.

**Comment by wbamberg** [comment]
Fair enough, I hadn't noticed that :). Let's keep it then.

File: live-examples/css-examples/svg/pointer-events.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<section id="example-choice-list" class="example-choice-list" data-property="animation">
```
Comment (wbamberg) [inline live-examples/css-examples/svg/pointer-events.html]: Should be "pointerEvents" ?

**Comment by ddbeck** [comment]
A quick question: should the `data-property` value be camel case or the same as the property itself? I'm a little mixed up across PRs about which is the right way to do it.

**Comment by wbamberg** [comment]
> should the data-property value be camel case or the same as the property itself? 

As I understand things, it doesn't matter. It's used here: https://github.com/mdn/interactive-examples/blob/master/js/editor-libs/mce-utils.js#L41, and as far as I can tell, either form works:

```
var a = document.createElement('div');

a.style["pointerEvents"]
< ""

a.style["pointer-events"]
< ""

a.style["bananas"] 
< undefined
```

But still, it's a shame we are not consistent here :(.

**Comment by ddbeck** [comment]
@wbamberg OK, I did a quick-and-dirty census of the `data-property` values with ripgrep:

```fish
$ rg 'data-property="[\w-]+"' | wc -l  # total
238

$ rg --case-sensitive 'data-property="[a-z-]+"' | wc -l # hyphenated
192

$ rg --case-sensitive 'data-property="\w+[A-Z]+\w+"' | wc -l # camelCase
46
```

So I guess I'll stick with hyphenated going forward. If you want, I can open an issue around making them consistent.

**Comment by wbamberg** [comment]
Thanks for this analysis, @ddbeck !

>  If you want, I can open an issue around making them consistent.

Yes, please! If you can get a list of the camelCase examples, that would be a really helpful thing to add. We should also make the docs on this clearer.

I expect this mechanism might change in the future, as I want us to have a better way of dealing with limited support (see also https://github.com/mdn/interactive-examples/issues/532 and https://github.com/mdn/interactive-examples/issues/534). But that might be a while away.

**Pull request merged**